README.md

```diff
@@ -49,6 +49,33 @@ All environments use the same Jenkinsfile, with parameters controlled in [Jenkin
 * "Push" - `docker push` - push changes to ECR.
 * "Bounce" - Update ECS service and wait until stable. Will fail if not stable after 10mins.
 
+### Database Migrations
+
+There are 2 databases per environment, both use EFCore and migrations:
+
+* DDS - `Wellcome.Dds.Repositories.DdsContext`
+* DDSInstrumentation - `Wellcome.Dds.AssetDomainRepositories.DdsInstrumentationContext`
+
+Database migrations are automatically applied on start of the Dashboard project for all environments _except Production_. This is controlled in dashboard startup:
+
+```cs
+if (!env.IsProduction())
+{
+    UpdateDatabase(logger);
+}
+```
+
+For production, the migrations need to be manually applied. Idempotent scripts can be generated by using the following commands (from `./src/Wellcome.Dds/`):
```

Comment:
Where and how do you _run_ these scripts?

Reply:
Updated with details of where to find connection string and need to use bastion.